rtcpeerconnection.go

```diff
@@ -1165,6 +1165,29 @@ func (pc *RTCPeerConnection) CreateDataChannel(label string, options *RTCDataCha
 		params.ID = *options.ID
 	}
 
+	if options != nil {
+		// Ordered indicates if data is allowed to be delivered out of order. The
+		// default value of true, guarantees that data will be delivered in order.
+		if options.Ordered != nil {
+			params.Ordered = *options.Ordered
+		}
+
+		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #7)
+		if options.MaxPacketLifeTime != nil {
+			params.MaxPacketLifeTime = options.MaxPacketLifeTime
+		}
+
+		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #8)
+		if options.MaxRetransmits != nil {
+			params.MaxRetransmits = options.MaxRetransmits
+		}
+
+		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #9)
+		if options.Ordered != nil {
+			params.Ordered = *options.Ordered
+		}
+	}
+
```

Comment:
Can you delete the commented out code below (since you are implementing these things now!) awesome work :)

Reply:
I did. v(--)